packages/docs/static/themes/cmsgov-theme.css

```diff
@@ -137,7 +130,7 @@
 --shadow-focus: inset 0 0 0 1px #262626;
 --shadow-focus-inverse: inset 0 0 0 1px #262626;
 --shadow-focus-link: 0 3px #262626;
---shadow-focus-link-inverse: 0 3px #7c6210;
+--shadow-focus-link-inverse: 0 3px undefined;
```

Comment:
Seems like we should remove these as well if they're not going to be used?

Reply:
Yeah, I think that's what Scott was saying above, but nobody seems to know where they're used. We should verify that they aren't used to re-create some of the styles in Sketch

Reply:
Updated and pushed a few commits